seltza/assignment3.py

```diff
@@ -0,0 +1,39 @@
+import turtle
+
+# Alex Carpenter
+# Colin Reilly School of Brogramming Assignment 3
+
+s = turtle.getscreen()
+
+t = turtle.Turtle()
+
+# t.right(90)
+# t.forward(100)
+# t.left(90)
+# t.backward(100)
```

Comment:
Heard you like comments, so here's a comment for your comments.

Reply:
Shut up please.

Reply:
We put a review in your comment so you can comment while you review

Reply:
All of you, shut up. Please.

Reply:
I am begging you all to read a book.